11ty/definitions/third-world-country.md

```diff
@@ -0,0 +1,20 @@
+---
+title: third-world country
+slug: third-world-country
+defined: true
+speech: noun
+---
+
+a problematic term used to refer to a country that doesn’t quite align with Western ideology or the Western image of society. (_Also common are terms such as “third-world” (adj.), “first-world”, “developing/developed nations”, etc._)
```

Comment:
```suggestion
describes a country that doesn’t quite align with Western ideology or the Western image of society. (_Also common are terms such as “third-world” (adj.), “first-world”, “developing/developed nations”, etc._)
```

Reply:
Totally makes sense — thank you for this!